src/cli/cli_mac_filter.cpp

```diff
@@ -288,6 +288,25 @@ template <> otError MacFilter::Process<Cmd("addr")>(Arg aArgs[])
     return error;
 }
 
+/**
+ * @cli macfilter rss
+ * @code
+ * macfilter rss
+ * 0f6127e33af6b403 : rss -95 (lqi 1)
+ * Default rss: -50 (lqi 3)
+ * Done
+ * @endcode
+ * @par
+ * Provides the following information:
+ * - Listing of all the extended addresses
+ * where the received signal strength (`rss`) has been set to be different from the default value.
+ * The link quality indicator (`lqi`) is also shown. The `rss` and `lqi`
+ * settings map to each other. If you set one, the value of the other gets set automatically.
+ * This list of addresses is called the `RssIn List`. Setting either the `rsi` or the `lqi`
+ * adds the corresponding extended address to the `RssIn` list.
```

Comment:
It probably wont matter for generated doxygen, but we can have lines wrap around same length here as well? (easier/nicer to read).
```suggestion
 * - Listing of all the extended addresses where the received signal strength (`rss`) has 
 *   been set to be different from the default value.  The link quality indicator (`lqi`) 
 *    ...
 *